bundle-workflow/tests/tests_test_workflow/test_integ_workflow/integ_test/test_integ_test_suite.py

```diff
@@ -52,7 +52,7 @@ def test_execute_with_multiple_test_configs(
         mock_system_execute.return_value = 200, "success", "failure"
         mock_local_test_cluster.create().__enter__.return_value = "localhost", "9200"
         mock_script_finder.return_value = "integtest.sh"
-        test_configs = integ_test_suite.execute()
+        integ_test_suite.execute()
```

Comment:
Check what it returns.

Reply:
Sure. Can you suggest any way to test `test_results` (https://github.com/opensearch-project/opensearch-build/pull/641/files#diff-ba2eae80a0960e4f0edf89515a1bc17cc748d943f5eb9f8a1e6155519aa8ca93R61) as it's an object inside a method of `TestResultsComponent`?

Reply:
I mean something like this:

```
test_results = integ_test_suite.execute()
self.assertEqual(test_results.status, 0)
```

Reply:
I see. Pushed the change